src/components/structures/MatrixChat.tsx

```diff
@@ -586,6 +586,7 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                 break;
             case 'logout':
                 dis.dispatch({action: "hangup_all"});
+                dis.dispatch({action: "logout"});
```

Comment:
am I missing something or is a logout dispatch firing another logout dispatch which would get stuck in a loop?
Why is this needed atop the dispatcher bus listener for logout in PersistedElement itself?

Reply:
You're completely right, that was the remainder of some of my testing that I forgot to clean up.
Thank you 💯